kafka_consumer/datadog_checks/kafka_consumer/kafka_consumer.py

```diff
@@ -21,12 +22,13 @@ class KafkaCheck(AgentCheck):
         'ssl_password': {'name': 'tls_private_key_password'},
     }
 
-    def __init__(self, name, init_config, instances):
+    def __init__(self, name, init_config, instances, client=KafkaPythonClient):
```

Comment:
Since we plan on eventually removing the code for `KafkaPythonClient` during the transition to the Confluent library, do we still need to do this?

Reply:
I see that this is primarily for testing, although I think we should instead write tests that would be library-independent and remove the library-dependent tests during the transition.

Reply:
Good point, I'll remove this for the PR